upup/models/cloudup/resources/addons/external-dns.addons.k8s.io/v0.3.0.yaml.template

```diff
@@ -0,0 +1,88 @@
+apiVersion: extensions/v1beta1
```

Comment:
We need to have the names with the correct k8s versioning. We have pre k8s 1.6 and k8s 1.6 naming.

Does this run on 1.5?